docs/docusaurus/docs/tutorials/quickstart/quickstart.md

```diff
@@ -30,6 +31,55 @@ Windows support for the open source Python version of GX is currently unavailabl
 - pip
 - An internet browser
 
+## Quickstart Workflow
+
+This quickstart walks you through the steps required to implement a simple, end-to-end GX workflow.
+
+```mermaid
+flowchart LR
+
+    subgraph Configure["<b>CONFIGURE</b>"]
+        Configure_A(Install\n<b>GX</b>):::gxWorkflowStep --> Configure_B("Create a\n<b>Data Context</b>"):::gxWorkflowStep
+    end
+
+    Configure_B --> Connect_A
+
+    subgraph Connect["<b>CONNECT</b>"]
+        Connect_A("Add a\n<b>Data Source</b>"):::gxWorkflowStep --> Connect_B("Add a\n<b>Data Asset</b>"):::gxWorkflowStep
+    end
+
+    Connect_B --> Define_A
+
+    subgraph Define["<b>DEFINE</b>"]
+        Define_A("Create a\n<b>Validator</b>"):::gxWorkflowStep --> Define_B("Add\n<b>Expectations</b>"):::gxWorkflowStep
+    end
+
+    Define_B --> Validate_A
+
+    subgraph Validate["<b>VALIDATE</b>"]
+        Validate_A("Run a\n<b>Checkpoint</b>"):::gxWorkflowStep --> Validate_B("View\n<b>ValidationResults</b>"):::gxWorkflowStep
+    end
+
+%% Define styles for workflow step nodes and container subgraphs.
+classDef gxWorkflowStep fill:#FF9B67,stroke-width:0px
+style Configure fill:#E6E7E8,stroke-width:0px
+style Connect fill:#E6E7E8,stroke-width:0px
+style Define fill:#E6E7E8,stroke-width:0px
+style Validate fill:#E6E7E8,stroke-width:0px
+
+%% Link each workflow step to underlying documentation.
+click Configure_A "https://docs.greatexpectations.io/docs/guides/setup/installation/install_gx"
+click Configure_B "https://docs.greatexpectations.io/docs/guides/setup/configuring_data_contexts/instantiating_data_contexts/instantiate_data_context/"
+click Connect_A "https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/connect_to_data_lp"
+click Connect_B "https://docs.greatexpectations.io/docs/guides/connecting_to_your_data/connect_to_data_lp"
+click Define_A "https://docs.greatexpectations.io/docs/terms/validator"
+click Define_B "https://docs.greatexpectations.io/docs/guides/expectations/expectations_lp"
+click Validate_A "https://docs.greatexpectations.io/docs/guides/validation/checkpoints/checkpoint_lp"
+click Validate_B "https://docs.greatexpectations.io/docs/terms/data_docs"
+```
+
+*Click on any individual workflow step to explore additional documentation for that step.*
```

Comment:
```suggestion
Click a workflow step to view related documentation.
```
![image](https://github.com/great-expectations/great_expectations/assets/104205257/3292e9e7-8899-40ef-b56c-4a25de95aa2e)
![image](https://github.com/great-expectations/great_expectations/assets/104205257/88384a10-06d9-4383-8f90-4e31254cfa21)

Reply:
I recommend incorporating this text into the introductory statement.

Reply:
Done!